### ISSUE TITLE
[bug] JSON schema unavailable

### ISSUE THREAD

**Issue by dewil-official** [inlang/monorepo#1366]
### Problem

Just upgraded from the old `.js` config file format to `inlang.project.json` but now I'm getting this error:

`The settings is invalid according to the schema.`

This is seemingly because the `$schema` that you guys provide in your docs and everywhere ([link](https://inlang.com/schema/project-config)) is returning a 404.

I also ran the linter locally to validate the config file and AFAIK everything should be correct.

### Expected behavior

The JSON schema should be available and the Inlang Web Editor shouldn't be failing because of that.

### Reproduction

- [Check our repo](https://inlang.com/editor/github.com/remnoteio/translation)
or
- [Check your schema link](https://inlang.com/schema/project-config)
or
- (Presumably this should also happen if you create a blank project according to your documentation, but haven't tested)

### Other information

- Chrome (latest)
- Windows 11
- VS Code
- Node v18 for running the CLI through npx
(probably none of that is relevant though)

**Comment by samuelstroschein** [comment]
@dewil-official wups, the CLI wasn't updated. We renamed `project-config` to `project-settings` and removed the nesting from the settings file. I opened https://github.com/remnoteio/translation/pull/24 which fixes your bug. 

Is everything else working as expected?


PS it might be that the browser is still caching an old lint rule. If you get "InvalidModuleExportError", try to clean the cache. Unfortunately, we can't clean the cache of the browser manually.
